### PR TITLE
Merge the verified-1:1 commits into main

### PR DESCRIPTION
**Pull request by Hocuri**
For the implementers of the other UIs:
- https://github.com/deltachat/deltachat-android/pull/2560 has a checklist what you have to do.
- https://github.com/deltachat/deltachat-android/pull/2621 has a screenshot
- I don't think you have to worry about the third commit in this PR here ([faeb634](https://github.com/deltachat/deltachat-android/pull/2752/commits/faeb6343628d84393940cf353abcc720408d41d4), which is the same as https://github.com/deltachat/deltachat-android/pull/2624), it just was a bug in Android

I used `git log main..feat/verified-1to1` to see all the commits that are missing on main and then cherry-picked the ones related to verified 1:1 chats.

Marking as a draft since verified 1:1 chats are missing in core stable, see my message in "Delta Chat Dev".

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -834,12 +835,11 @@
     <string name="ephemeral_timer_weeks_by_you">You set disappearing messages timer to %1$s weeks.</string>
     <!-- %1$s will be replaced by the number of weeks (always >1) the timer is set to, %2$s will be replaced by name and address of the contact -->
     <string name="ephemeral_timer_weeks_by_other">Disappearing messages timer set to %1$s weeks by %2$s.</string>
-    <string name="protection_enabled_by_you">You enabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_enabled_by_other">Chat protection enabled by %1$s.</string>
-    <string name="protection_disabled_by_you">You disabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_disabled_by_other">Chat protection disabled by %1$s.</string>
+    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```
Comment (r10s) [inline res/values/strings.xml:838]:
```suggestion
    <!-- this may be shown instead of the chat's input field, with the buttons as "More Info" or "OK" -->
    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```

Comment (Hocuri) [reply res/values/strings.xml:838]:
```suggestion
    <!-- this may be shown instead of the chat's input field, with buttons "More Info" and "OK" -->
    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```

File: src/org/thoughtcrime/securesms/ConversationFragment.java [hunk 2]
```diff
@@ -856,7 +856,7 @@ public void onQuoteClicked(DcMsg messageRecord) {
       public void onShowFullClicked(DcMsg messageRecord) {
         Intent intent = new Intent(getActivity(), FullMsgActivity.class);
         intent.putExtra(FullMsgActivity.MSG_ID_EXTRA, messageRecord.getId());
-        intent.putExtra(FullMsgActivity.IS_CONTACT_REQUEST, getListAdapter().getChat().isContactRequest());
+        intent.putExtra(FullMsgActivity.BLOCK_LOADING_REMOTE, getListAdapter().getChat().isHalfBlocked());
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ConversationFragment.java:859]: was just wondering about loading remote image blocking in iOS. if in doubt, it probably makes sense to mimic behaviour of contact-request here

**Review by r10s — APPROVED** [review]

File: src/org/thoughtcrime/securesms/connect/DcHelper.java [hunk 3]
```diff
@@ -438,4 +447,20 @@ public static String getConnectivitySummary(Context context, int connectedString
...
+            .setMessage(context.getString(R.string.chat_protection_broken_explanation, name))
+            .setNeutralButton(R.string.learn_more, (d, w) -> IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/684/en/help#verificationbroken"))
+            .setNegativeButton(R.string.qrscan_title, (d, w) -> context.startActivity(new Intent(context, QrActivity.class)))
+            .setPositiveButton(R.string.ok, null)
+            .setCancelable(true)
+            .show();
+  }
+
+  public static void showProtectionEnabledDialog(Context context) {
+    IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/684/en/help#whatdoesverifiedmean");
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/connect/DcHelper.java:462]: i think, we should show an `AlertDialog` as for `showVerificationBrokenDialog()`, just to have the same and not unexpected behaviour. (maybe i argued the other way round at some point, but using it now, i think, this is better:)

but that can also go to another pr.

File: src/org/thoughtcrime/securesms/connect/DcHelper.java [hunk 4]
```diff
@@ -438,4 +447,20 @@ public static String getConnectivitySummary(Context context, int connectedString
           return context.getString(R.string.connectivity_not_connected);
       }
   }
+
+  public static void showVerificationBrokenDialog(Context context, String name) {
+    new AlertDialog.Builder(context)
+            .setMessage(context.getString(R.string.chat_protection_broken_explanation, name))
+            .setNeutralButton(R.string.learn_more, (d, w) -> IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/684/en/help#verificationbroken"))
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/connect/DcHelper.java:454]:
```suggestion
            .setNeutralButton(R.string.learn_more, (d, w) -> IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/733/en/help#verificationbroken"))
```

File: src/org/thoughtcrime/securesms/connect/DcHelper.java (hunk 3 above)
Comment (r10s) [inline src/org/thoughtcrime/securesms/connect/DcHelper.java:462]:
```suggestion
    IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/733/en/help#whatdoesverifiedmean");
```

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/connect/DcHelper.java:462]:
```suggestion
    IntentUtils.showBrowserIntent(context, "https://staging.delta.chat/733/en/help#verifiedchats");
```

File: res/values/strings.xml [hunk 5]
```diff
@@ -834,12 +834,13 @@
...
-    <string name="protection_enabled_by_you">You enabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_enabled_by_other">Chat protection enabled by %1$s.</string>
-    <string name="protection_disabled_by_you">You disabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_disabled_by_other">Chat protection disabled by %1$s.</string>
+    <!-- this may be shown instead of the chat's input field, with buttons "More Info" and "OK" -->
+    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
+    <string name="chat_protection_enabled_tap_to_learn_more">Messages are guaranteed to be end-to-end encrypted from now on. Tap to learn more.</string>
+    <string name="chat_protection_enabled_explanation">It is now guaranteed that all messages in this chat are end-to-end encrypted.\n\nEnd-to-end encryption keeps your personal messages between you and your chat partners. Not even your email server can read them.</string>
```
Comment (r10s) [inline res/values/strings.xml:840]:
```suggestion
    <string name="chat_protection_enabled_explanation">It is now guaranteed that all messages in this chat are end-to-end encrypted.\n\nEnd-to-end encryption keeps your personal messages between you and your chat partners private. Not even your email server can read them.</string>
```

keeps what? i think there is a "private" missing?

**Review by r10s — APPROVED** [review]
